demos/ios/MASVS-RESILIENCE/MASTG-DEMO-0023/MASTG-DEMO-0023.md

```diff
@@ -0,0 +1,32 @@
+---
+platform: ios
+title: Verify App is Debuggable with r2
+code: [swift]
+id: MASTG-DEMO-0023
+test: MASTG-TEST-0082
+---
+
+### Sample
+
+The code snippet below shows sample code that verify the application is debuggable.
+
+{{ MastgTest.swift }}
```

Comment:
Instead of this we need the modified entitlements file, so that we can build the app using it, and the reversed version.

Here's the original: https://github.com/cpholguera/MASTestApp-iOS/blob/main/entitlements.plist

```suggestion
{{ entitlements.plist # entitlements_reversed.plist }}
```